docs/cache/develop/integrations/drupal-integration.md

```diff
@@ -7,6 +7,16 @@ pagination_next: null
 ---
 
 # Integrating Momento Cache with Drupal
+# METHOD 1: (Recommended)
```

Comment:
We should completely deprecate/disappear the module hosted in our repository, so there should only be one method for integrating with Drupal. I think it would make more sense to just remove the old content from this file. It looks like everything in here is in the README you linked to so you shouldn't need to preserve any of it.